src/main/java/org/spongepowered/api/command/args/CommandArgs.java

```diff
@@ -153,6 +157,20 @@ public String getRaw() {
     }
 
     /**
+     * Insert an {@link Iterable} of args as the next args to be
+     * returned by {@link #next()}.
+     *
+     * @param values The arguments to insert
+     */
+    public void insertArgs(Iterable<String> values) {
```

Comment:
Should this be called addAll and take an index to match the java collections API? Or does insertArgs work just as well?